.github/scripts/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+echo "Setup OLM"
+$HOME/.travis/setup-olm.sh
```

Comment:
I don't think minikube action if good fit for this. Because if we left minikube action as it is, other teams like streams,.. can use it as well :) I will just get this outside of build step.